test/common/upstream/ring_hash_lb_test.cc

```diff
@@ -320,7 +322,8 @@ TEST_P(DISABLED_RingHashLoadBalancerTest, DetermineSpread) {
   config_.value(envoy::api::v2::Cluster::RingHashLbConfig());
   config_.value().mutable_minimum_ring_size()->set_value(min_ring_size);
   config_.value().mutable_deprecated_v1()->mutable_use_std_hash()->set_value(false);
-  RingHashLoadBalancer thread_aware_lb{priority_set_, stats_, runtime_, random_, config_};
+  RingHashLoadBalancer thread_aware_lb{priority_set_, stats_,  runtime_,
+                                       random_,       config_, common_config_};
```

Comment:
Is there weird extra whitespace here?  Who are you and what have you done with Matt?  ;-)

Reply:
clang-format. I don't like it either but not much we can do about it.